bitnami/spark/README.md

```diff
@@ -0,0 +1,256 @@
+| `worker.readinessProbe.enabled`             | Turn on and off readiness probe                                                             | `true`                                                  |
+| `worker.readinessProbe.initialDelaySeconds` | Delay before liveness probe is initiated                                                    | 5                                                       |
+| `worker.readinessProbe.periodSeconds`       | How often to perform the probe                                                              | 10                                                      |
+| `worker.readinessProbe.timeoutSeconds`      | When the probe times out                                                                    | 5                                                       |
+| `worker.readinessProbe.failureThreshold`    | Minimum consecutive failures for the probe to be considered failed after having succeeded.  | 6                                                       |
+| `worker.readinessProbe.successThreshold`    | Minimum consecutive successes for the probe to be considered successful after having failed | 1                                                       |
+| `security.passwordsSecretName`              | Secret to use when using security configuration to set custom passwords                     | No default                                              |
+| `security.rpc.authenticationEnabled`        | Enable the RPC authentication                                                               | `no`                                                    |
+| `security.rpc.encryptionEnabled`            | Enable the encryption for RPC                                                               | `no`                                                    |
+| `security.storageEncryptionEnabled`         | Enable the encryption of the storage                                                        | `no`                                                    |
+| `security.ssl.enabled`                      | Enable the SSL configuration                                                                | `no`                                                    |
+| `security.ssl.needClientAuth`               | Enable the client authentication                                                            | `no`                                                    |
+| `security.ssl.protocol`                     | Set the SSL protocol                                                                        | `TLSv1.2`                                               |
+| `security.certificatesSecretName`           | Set the name of the secret that contains the certificates                                   | No default                                              |
```

Comment:
I am not sure about how this is handled. I should be able to include my own secret and, if that is the case, then do not generate a random one.

Reply:
That is how it works. If you do not provide a secret name with passwords a random one is generated. But the certificates can't be auto-generated.